### ISSUE TITLE
aerich upgrade doesn't finish

### ISSUE THREAD

**Issue by manzato** [tortoise/aerich#131]
OS: Ubuntu 20.04.2 LTS
Python: 3.8.5
Aerich: 0.5.0

I'm trying to run the migrations when my software starts by running `aerich upgrade` but control never returns (the process doesn't exit). Is this expected? Doesn't sound like, but may be. I did some debugging but couldn't find something obvious.

Log:
```
(app) manzato@x2:/app/src$ aerich upgrade
No upgrade items found
```

**Comment by bhch** [comment]
I'm also seeing a similar freezing while `init-db`. This seems related to #63.

**Comment by long2ice** [comment]
Fixed in v0.5.1

**Comment by manzato** [comment]
Thanks for the quick turn-around!
